.github/workflows/release_tests.yml

```diff
@@ -32,3 +32,8 @@ jobs:
     - name: Canary test public API
       run: |
         python -m pytest -r sx tests/test_public_api.py
+    - name: Verify requirements in codemeta.json
+      run: |
+        python -m pip install jq "codemetapy>=0.3.4"
+        codemetapy --no-extras pyhf > codemeta_generated.json
+        diff <(jq -S .softwareRequirements codemeta_generated.json) <(jq -S .softwareRequirements codemeta.json)
```

Comment:
Taken from https://stackoverflow.com/questions/31930041/using-jq-or-alternative-command-line-tools-to-compare-json-files